styles/app.css

```diff
@@ -0,0 +1,70 @@
+
+* {
+  box-sizing: border-box;
+}
+
+ *::before. *::after {
+   box-sizing: inherit;
+ }
+
+body {
+margin: 0;
+  }
+
+.grid {
+margin: .5rem;
+padding: 1rem;
+border: 1px solid #D2D2D2;
+  }
+
+.column1, .column2, .column3 {
+width: 32%;
+padding-left: 0.6rem;
+border: 1px solid #D2D2D2;
+border-radius: 3px;
+}
+
+.column1 {
+float: left;
+}
+
+.column2 {
+display: inline-block;
+margin-left: 1rem;
+}
+
+.column3 {
+float: right;
+}
+
+img {
+height: 26rem;
```

Comment:
Don't set the `height` of these images. 

What you want is for the large octocat to fill the block side to side, so use that measurement instead and use a percentage to make sure it scales over time.